workflow/hosts/hera.yaml

```diff
@@ -1,30 +1,32 @@
-BASE_GIT: '/scratch1/NCEPDEV/global/glopara/git'
+# Paths
 DMPDIR: '/scratch1/NCEPDEV/global/glopara/dump'
+BASE_GIT: '/scratch1/NCEPDEV/global/glopara/git'
 BASE_DATA: '/scratch1/NCEPDEV/global/glopara/data'
 BASE_IC: '/scratch1/NCEPDEV/global/glopara/data/ICSDIR'
+AERO_INPUTS_DIR: /scratch1/NCEPDEV/global/glopara/data/GEFS_ExtData/20250310
 PACKAGEROOT: '/scratch1/NCEPDEV/global/glopara/nwpara'
-COMINsyn: '/scratch1/NCEPDEV/global/glopara/com/gfs/prod/syndat'
 HOMEDIR: '/scratch1/NCEPDEV/global/${USER}'
```

Comment:
now that we are Jinja'ing, could we do something like this:
```
HOMEDIR: "{{ GW_HOMEDIR | default('/scratch1/NCEPDEV/global/${USER}') }}"
```
so that power users can set more of these things (stmp, ptmp, etc.)

Reply:
The yamls `under `hosts/` are presently not rendered by jinja.  They are used to construct the dictionary for parsing jinja configs under `parm/config`.

We can add this feature request to our backlog for future work.